Gemfile

```diff
@@ -70,14 +70,14 @@ gem 'ruby-saml'
 gem 'safe_target_blank', '>= 1.0.2'
 gem 'saml_idp', github: '18F/saml_idp', tag: '0.21.0-18f'
 gem 'scrypt'
-gem 'simple_form', '>= 5.0.2'
+gem 'simple_form', '>= 5.3.1'
```

Comment:
this is a big jump, I would be curious to see if we could avoid upgrading this?

Reply:
Yeah these upgrades look off. I would expect we'd only need to bump Rails, and there should also be corresponding `Gemfile.lock` changes included.